esti/s3_gateway_test.go

```diff
@@ -48,7 +48,7 @@ func TestS3UploadAndDownload(t *testing.T) {
 	for _, sig := range sigs {
 		t.Run("Sig"+sig.Name, func(t *testing.T) {
 			// Use same sequence of pathnames to test each sig.
-			rand := rand.New(rand.NewSource(17))
+			randomizer := rand.New(rand.NewSource(17))
```

Comment:
This name confuses me. It's a Rand, a randomizer is something else
I'd be happier with r,  or if that's too short for comfort perhaps prng. Or rand, really...

Reply:
Done, went with 'r'.